Dockerfile

```diff
@@ -3,7 +3,7 @@ ENV RAILS_VERSION 7.0.3.1
 
 RUN apt-get update -qq && apt-get install -y nodejs npm
 # Bugfix for https://github.com/rubyjs/mini_racer/issues/220#issuecomment-1010724771
-RUN gem update --system
+RUN gem update --system 3.4.2
```

Comment:
Typo!
```suggestion
RUN gem update --system 3.4.22
```

Reply:
Thanks, your suggestion works well in my local environment.
I add your suggestion.